libs/ssl-config/src/main/java/org/elasticsearch/common/ssl/SslKeyConfig.java

```diff
@@ -31,5 +36,27 @@
      */
     X509ExtendedKeyManager createKeyManager();
 
+    /**
+     * @return A list of private keys and their associated certificates
+     */
+    List<Tuple<PrivateKey, X509Certificate>> getKeys();
+
+    /**
+     * @return A collection of {@link X509Certificate certificates} used by this config.
+     */
+    Collection<? extends StoredCertificate> getConfiguredCertificates();
```

Comment:
Do we foresee exteding StoredCertificate ? Can we make the return type non generic for now ?